bootstrap.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/*
+ * This file is part of the MediaModule for Zikula.
+ *
+ * (c) Christian Flach <hi@christianflach.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
They are still there, but not always called. They are called for controllers (of course) and block handlers, but for example not for content types or searches. But at least it seems cleaner to include_once this bootstrap file at some places instead of something inside the vendor folder - especially in case something else is going to be added in future maybe.